charts/geoserver/values.yaml

```
@@ -0,0 +1,81 @@
+# Default values for geoserver.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: docker.osgeo.org/geoserver
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+username: admin
```

can we do something like
```suggestion
auth:
  username: admin
  # use an existing secret if specified
  # existingSecret: geoserver
  # passwordKey: geoserver-password
  password: geoserver
```

---

implemented